src/docs/uxd/actionsOnSystems/DOC.md

```diff
@@ -0,0 +1,34 @@
+# RHCS - Actions on systems
+
+For a system, there are 2 primary types of actions:
+
+- Service-specific actions: e.g. "Opt-out of vulnerability reporting"
+- Global system actions: e.g. "Update host name"
+
+## Service view of a system
+
+Service-specific actions should be available from the view of the system from that service.
+Global system actions are not available.
+Service-specific actions can be inline, or in the header, or related to the table. (Anything that is related to the service!)
+_**Why?** Keep the actions in context of what the user is doing._
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/serviceSpecificActions.png?raw=true "serviceSpecificActions")
+*Service view of a system, showing 2 service-specific actions (Remediate with Ansible, Opt-out of vulnerability reporting)*
+
+## Inventory view of a system
+
+Global system actions should be available from the inventory. (Not on service pages).
+Service-specific actions should also be available in the inventory.
+_**Why?** Inventory is global. It is the comprehensive view of a system & everything that is related to it. Therefore, should have all the actions available to that system._
+
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/inventoryInlineGlobal.png?raw=true "inventoryInlineGlobal")
```

Comment:
Alt text should be defined, not just alt text. 
So, for example, "Image of an inventory view which shows inline global actions for [x] and [y]"

Reply:
Soooo if you were looking at [this resource](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images) (which is otherwise good) it really doesn't explain the alt text concept. 
In the markdown syntax, the text in the square brackets for an image translates to be the "alt=blah blah" on the img tag in HTML. 

Honestly, its not a huge deal for this particular project, but for anything that is public facing its important to write good alt text for accessibility (aka people who can't see images can read the text, or people who only have access to slow internet can read it too so they don't need to download images over restricted or slow data)